pandas/core/arrays/datetimelike.py

```diff
@@ -1397,6 +1397,24 @@ def _time_shift(
         #  to be passed explicitly.
         return self._generate_range(start=start, end=end, periods=None, freq=self.freq)
 
+    def _accumulate(self, name: str, *, skipna: bool = True, **kwargs):
+
+        data = self._data.copy()
+
+        if name in {"cummin", "cummax"}:
+            func = np.minimum.accumulate if name == "cummin" else np.maximum.accumulate
```

Comment:
do the numpy functions not work directly?

Reply:
This has a different behavior than going through nanops. The initial pr included this but I ripped it out to keep it a bit more focused. Plan to tackle this afterwards, but we have to decide what we actually want here first. Hence only doing masked ops here

Reply:
> This has a different behavior than going through nanops

can you expand on this?

Reply:
skipna is the problem, difference in behavior with regards to floats, remembered this incorrect.

See https://github.com/pandas-dev/pandas/pull/28509#issuecomment-778741118

Wanted to investigate this as a follow up when we can check this more isolated